contracts/session/session_manager.go

```diff
@@ -0,0 +1,12 @@
+package session
+
+type Manager interface {
+	// BuildSession constructs a new session with the given handler and session ID.
+	BuildSession(handler Handler, id ...string) Session
+	// Driver retrieves the session driver by name.
+	Driver(name ...string) (Handler, error)
+	// Extend extends the session manager with a custom driver.
+	Extend(driver string, handler func() Handler) Manager
+	// Store retrieves a Session by session ID.
+	Store(sessionId ...string) Session
```

Comment:
We require the `Store` method. In Laravel, the Manager class inherits all methods of `Store`, eliminating the need to create it separately; it can be easily utilized with facades. Thus, within the session facade, if we intend to manage a particular session, we can either create a distinct facade or simply add the `Store` method, which will either create a new session or retrieve an existing one.
Ref: https://github.com/laravel/framework/blob/10.x/src/Illuminate/Support/Manager.php
![image](https://github.com/goravel/framework/assets/84431594/f24ae13b-b3de-40d4-bb33-ce837115b5c0)

Reply:
I have remove `Store` method from `Manager`.